### ISSUE TITLE
NullPointerException errors on connect

### ISSUE THREAD

**Issue by alex-karo** [AntonShuvaev/elasticsearch4idea#93]
I get errors when connecting to one instance the first time.
I get six `java.lang.NullPointerException` in Notifications without any additional info.
If I go to "Connection Properties", click "Test Connection", and then click Ok - the plugin connects successfully.

1) If I stop the connection and connect again - it works well.
2) I get these errors again after reloading IDEA
3) I cannot reproduce it for other instances of ES
4) It is reproduced at least 2 of my laptops

macOS 12.6.2 (21G320)
IntelliJ IDEA 2022.3.1 (Ultimate Edition)
Build #IU-223.8214.52, built on December 20, 2022
Elasticsearch plugin 2022.3.3-223
Elasticsearch 7.10, AWS, connection via credentials file

**Comment by AntonShuvaev** [comment]
@alex-karo, thanks for reporting this bug. Can you please enable additional logging and share stacktrace? Go to `Help | Diagnostic Tools | Debug Log Settings` and add `org.elasticsearch4idea:trace` to the textarea. Then restart IntelliJ to cause the error. When error occurs open `idea.log` (`Help | Show Log in Finder`) and find the stacktrace.

**Comment by alex-karo** [comment]
```
2023-01-20 17:18:37,599 [  71128]   FINE - #org.elasticsearch4idea.je - 
java.lang.NullPointerException
	at java.base/java.util.Objects.requireNonNull(Objects.java:208)
	at java.base/sun.nio.fs.UnixFileSystem.getPath(UnixFileSystem.java:263)
	at java.base/java.nio.file.Path.of(Path.java:147)
	at java.base/java.nio.file.Paths.get(Paths.java:69)
	at org.elasticsearch4idea.ji.a(SourceFile:103)
	at org.elasticsearch4idea.ji.a(SourceFile:58)
	at org.elasticsearch4idea.jh$e.invoke(SourceFile:1060)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at org.elasticsearch4idea.jh.a(SourceFile:61)
	at org.elasticsearch4idea.jh.a(SourceFile:57)
	at org.elasticsearch4idea.jh$i.invoke(SourceFile:292)
	at org.elasticsearch4idea.r.a(SourceFile:24)
	at org.elasticsearch4idea.r.b(SourceFile:63)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:243)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:29)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.executeFirstTaskAndHelpQueue(BoundedTaskExecutor.java:216)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:212)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:205)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-20 17:18:37,599 [  71128]   FINE - #org.elasticsearch4idea.je - 
java.lang.NullPointerException
	at java.base/java.util.Objects.requireNonNull(Objects.java:208)
	at java.base/sun.nio.fs.UnixFileSystem.getPath(UnixFileSystem.java:263)
	at java.base/java.nio.file.Path.of(Path.java:147)
	at java.base/java.nio.file.Paths.get(Paths.java:69)
	at org.elasticsearch4idea.ji.a(SourceFile:103)
	at org.elasticsearch4idea.ji.a(SourceFile:58)
	at org.elasticsearch4idea.jh$e.invoke(SourceFile:1060)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at org.elasticsearch4idea.jh.a(SourceFile:61)
	at org.elasticsearch4idea.jh.a(SourceFile:57)
	at org.elasticsearch4idea.jh$i.invoke(SourceFile:292)
	at org.elasticsearch4idea.r.a(SourceFile:24)
	at org.elasticsearch4idea.r.b(SourceFile:63)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:243)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:29)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.executeFirstTaskAndHelpQueue(BoundedTaskExecutor.java:216)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:212)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:205)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-20 17:18:37,599 [  71128]   FINE - #org.elasticsearch4idea.je - 
java.lang.NullPointerException
	at java.base/java.util.Objects.requireNonNull(Objects.java:208)
	at java.base/sun.nio.fs.UnixFileSystem.getPath(UnixFileSystem.java:263)
	at java.base/java.nio.file.Path.of(Path.java:147)
	at java.base/java.nio.file.Paths.get(Paths.java:69)
	at org.elasticsearch4idea.ji.a(SourceFile:103)
	at org.elasticsearch4idea.ji.a(SourceFile:58)
	at org.elasticsearch4idea.jh$e.invoke(SourceFile:1060)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at org.elasticsearch4idea.jh.a(SourceFile:61)
	at org.elasticsearch4idea.jh.a(SourceFile:57)
	at org.elasticsearch4idea.jh$i.invoke(SourceFile:292)
	at org.elasticsearch4idea.r.a(SourceFile:24)
	at org.elasticsearch4idea.r.b(SourceFile:63)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:243)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:29)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.executeFirstTaskAndHelpQueue(BoundedTaskExecutor.java:216)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:212)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:205)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-20 17:18:37,599 [  71128]   FINE - #org.elasticsearch4idea.je - 
java.lang.NullPointerException
	at java.base/java.util.Objects.requireNonNull(Objects.java:208)
	at java.base/sun.nio.fs.UnixFileSystem.getPath(UnixFileSystem.java:263)
	at java.base/java.nio.file.Path.of(Path.java:147)
	at java.base/java.nio.file.Paths.get(Paths.java:69)
	at org.elasticsearch4idea.ji.a(SourceFile:103)
	at org.elasticsearch4idea.ji.a(SourceFile:58)
	at org.elasticsearch4idea.jh$e.invoke(SourceFile:1060)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at org.elasticsearch4idea.jh.a(SourceFile:61)
	at org.elasticsearch4idea.jh.a(SourceFile:57)
	at org.elasticsearch4idea.jh$i.invoke(SourceFile:292)
	at org.elasticsearch4idea.r.a(SourceFile:24)
	at org.elasticsearch4idea.r.b(SourceFile:63)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:243)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:29)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.executeFirstTaskAndHelpQueue(BoundedTaskExecutor.java:216)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:212)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:205)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-20 17:18:37,599 [  71128]   FINE - #org.elasticsearch4idea.je - 
java.lang.NullPointerException
	at java.base/java.util.Objects.requireNonNull(Objects.java:208)
	at java.base/sun.nio.fs.UnixFileSystem.getPath(UnixFileSystem.java:263)
	at java.base/java.nio.file.Path.of(Path.java:147)
	at java.base/java.nio.file.Paths.get(Paths.java:69)
	at org.elasticsearch4idea.ji.a(SourceFile:103)
	at org.elasticsearch4idea.ji.a(SourceFile:58)
	at org.elasticsearch4idea.jh$e.invoke(SourceFile:1060)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at org.elasticsearch4idea.jh.a(SourceFile:61)
	at org.elasticsearch4idea.jh.a(SourceFile:57)
	at org.elasticsearch4idea.jh$i.invoke(SourceFile:292)
	at org.elasticsearch4idea.r.a(SourceFile:24)
	at org.elasticsearch4idea.r.b(SourceFile:63)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:243)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:29)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.executeFirstTaskAndHelpQueue(BoundedTaskExecutor.java:216)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:212)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:205)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-20 17:18:37,600 [  71129]   FINE - #org.elasticsearch4idea.je - 
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073)
	at org.elasticsearch4idea.jx$p.invoke(SourceFile:1158)
	at org.elasticsearch4idea.r.a(SourceFile:24)
	at org.elasticsearch4idea.r.b(SourceFile:49)
	at org.elasticsearch4idea.s$a.run(SourceFile:27)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:423)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:114)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsynchronously$6(CoreProgressManager.java:474)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:252)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:188)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:589)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:664)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:620)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:588)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:175)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:252)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NullPointerException
	at java.base/java.util.Objects.requireNonNull(Objects.java:208)
	at java.base/sun.nio.fs.UnixFileSystem.getPath(UnixFileSystem.java:263)
	at java.base/java.nio.file.Path.of(Path.java:147)
	at java.base/java.nio.file.Paths.get(Paths.java:69)
	at org.elasticsearch4idea.ji.a(SourceFile:103)
	at org.elasticsearch4idea.ji.a(SourceFile:58)
	at org.elasticsearch4idea.jh$e.invoke(SourceFile:1060)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at org.elasticsearch4idea.jh.a(SourceFile:61)
	at org.elasticsearch4idea.jh.a(SourceFile:57)
	at org.elasticsearch4idea.jh$i.invoke(SourceFile:292)
	at org.elasticsearch4idea.r.a(SourceFile:24)
	at org.elasticsearch4idea.r.b(SourceFile:63)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:243)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:29)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.executeFirstTaskAndHelpQueue(BoundedTaskExecutor.java:216)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:212)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:205)
	... 7 more
```

**Comment by AntonShuvaev** [comment]
Thanks, I will fix asap.

**Comment by AntonShuvaev** [comment]
Fixed in 2022.3.4
